README.md

```diff
@@ -20,48 +20,56 @@ npm install --production
 
 ## Configure your instance
 
-You will need to tell the Legislation Explorer server where your OpenFisca API instance can be reached, as well as where your repository resides for contributors to be directed to. This is done with a configuration file.
+You will need to tell the Legislation Explorer server where your OpenFisca API instance can be reached, as well as where your repository resides for contributors to be directed to. This is all done through environment variables, allowing you to use the same code for any instance by [injecting these elements at runtime](https://12factor.net/config).
 
-This config file needs to expose a JavaScript object with the following properties:
+To set these options, you need to define them by prefixing the `npm start` or `npm run dev` commands with their definitions, in the `$NAME=$VALUE` syntax.
+
+- `API_URL`: the URL at which your OpenFisca API root endpoint can be found. For example: `https://openfisca-aotearoa.herokuapp.com`. Defaults to `http://0.0.0.0:5000`.
+- `CHANGELOG_URL`: the URL at which the changelog for the country package can be found. Used on 404 pages. For example: `https://github.com/openfisca/openfisca-tunisia/blob/master/CHANGELOG.md`.
+
+> If it gets lengthy or you want to persist these values, you can also use a `.env` file.
+> Such a file is a plaintext file with name `.env` stored in the root directory of your legislation explorer instance (i.e. next to the `package.json` file). List all of your environment variables in the `$NAME=$VALUE` syntax, one per line. If you have trouble writing this file, read the [parsing rules](https://github.com/motdotla/dotenv#rules).
+> An example file is provided as `.env.example`, setting default values. You can copy it, but changing it won't have any impact: only a file named `.env` will be taken into account. You should not commit this file: it depends on each instance.
```

Comment:
I hoped this would have been clear enough with the reference to https://12factor.net/config and this:

> You should not commit this file: it depends on each instance.

If we use the `.env` file to store default values, then all clones will have changes that will conflict whenever there is a config (default value, renaming…) change.

How could I improve the documentation here?

Reply:
I'm sorry, I wan't very precise in my comment.

I think the documentation is great and very clear. My request would be that the `.env.example` (and not the `.env` as I hastily wrote) file contain the available environment variable with their default value.

Reply:
From previous experience as a user in similar situations, I liked having a `.env.example` containing all environment variables I could set, (possibly commented for the ones that were a little specific/uncommon). 
It made my life easier while configuring. My 2 currency-cents ;)